fluentd/generate_throttle_configs.rb

```diff
@@ -102,8 +102,9 @@ def create_default_docker(excluded)
 
   file_name = "/etc/fluent/configs.d/input/docker/default-docker.conf"
 
-  File.open(file_name, 'w') { |file|
-    file.write(<<-CONF)
+  unless ENV['USE_JOURNAL'] == "true"
```

Comment:
Can we just not create any throttle configs in the case where USE_JOURNAL == true?  e.g. move this check to here? https://github.com/richm/origin-aggregated-logging/blob/11e11e93caec61914ab853db912ebcc928ad318c/fluentd/generate_throttle_configs.rb#L241